src/ipfs.rs

```diff
@@ -0,0 +1,76 @@
+use libipld::{
+    cbor::DagCborCodec,
+    cid::{
+        multihash::{Code, MultihashDigest},
+        Cid,
+    },
+    codec::Codec,
+    Ipld,
+};
+use reqwest::multipart;
+use serde_json;
+
+// Store IPLD data on IPFS and get its CID
+pub async fn dag_put(host: &String, dag: Ipld) -> Result<String, reqwest::Error> {
```

Comment:
This is just making an Http request, essentially. Part of our Milestone 2 was adding this functionality, so you can store Lurk objects (encoded as IPLD, encoded as CBOR) on an ipfs node. Happy to factor it out into a separate crate if that's better

Reply:
Yes, as I mentioned to @samuelburnham elsewhere (I think) this doesn't really belong in `lurk-rs`.

I think for now it's fine to put in `fcomm` so it can be used there — especially if done in such a way that it will be easy to extract if/when we want to.